Emby.Drawing/ImageProcessor.cs

```diff
@@ -196,7 +199,7 @@ public bool SupportsTransparency(string path)
 
             var originalImagePath = originalImage.Path;
             var dateModified = originalImage.DateModified;
-            var originalImageSize = originalImage.Width > 0 && originalImage.Height > 0 ? new ImageSize(originalImage.Width, originalImage.Height) : (ImageSize?)null;
```

Comment:
Can this code be refactored to not use the <test> ? <if> : <else> construct? It was hard to read before, and with the new addition it's even harder to read. :)

Reply:
👍